lib/tutorials/getting-started.md

```diff
@@ -106,7 +106,10 @@ server.register({
     options: {
         reporters: [{
             reporter: require('good-console'),
-            args:[{ log: '*', response: '*' }]
+            events: {
+                request: '*',
```

Comment:
It was changed accidentally. Copy-pasted the code from the release notes (https://github.com/hapijs/good/issues/323) to test it locally and then probably forgot to update it when sent the PR. Created a new one: https://github.com/hapijs/hapijs.com/pull/169